src/test/feature/catalog/test_create_table.cpp

```diff
@@ -22,26 +22,26 @@ class TestCreateTable : public ::testing::Test {
 TEST_F(TestCreateTable, TestCreateTable1) {
   hawq::test::SQLUtility util;
   // prepare
-  util.execute("DROP TABLE IF EXISTS aggtest");
```

Comment:
The easier way to do this is creating a new schema `X`, set the default search_path to `X`. At the end of the test, drop the schema. In this way, you don't need to take care of dropping tables when you add new test cases. This is how GPDB is doing.